From 4fb8001507f92fc2596a2444ca64d779e86ccb6b Mon Sep 17 00:00:00 2001
From: Hongzhang Yang <Hongzhang.Yang@freescale.com>
Date: Mon, 6 Aug 2012 11:27:23 +0800
Subject: [PATCH 211/280] ENGR00219462 Update android_vpu_upgrades.sh to
 package i.MX5/6 firmware binaries

Update android_vpu_upgrades.sh to package i.MX5/6 firmware binaries

Signed-off-by: Hongzhang Yang <Hongzhang.Yang@freescale.com>
---
 vpu/android_vpu_upgrades.sh | 19 ++++++++++++++++---
 1 file changed, 16 insertions(+), 3 deletions(-)

diff --git a/vpu/android_vpu_upgrades.sh b/vpu/android_vpu_upgrades.sh
index 3c4fb68..492a595 100755
--- a/vpu/android_vpu_upgrades.sh
+++ b/vpu/android_vpu_upgrades.sh
@@ -461,9 +461,22 @@ if [ "$FIRMWARE_TO" != "none" ]; then
 	fi
 
 	cd $root
-	firmware_list="external/linux-firmware-imx/firmware/vpu/vpu_fw_imx51.bin
-		       external/linux-firmware-imx/firmware/vpu/vpu_fw_imx53.bin
-		       external/linux-firmware-imx/firmware/vpu/Android.mk"
+	firmware_list=""
+	ls external/linux-firmware-imx/firmware/vpu/vpu_fw_imx5*.bin;
+	if [ $? = 0 ];then
+		file=`ls external/linux-firmware-imx/firmware/vpu/vpu_fw_imx5*.bin`
+		firmware_list="$firmware_list $file"
+	fi
+	ls external/linux-firmware-imx/firmware/vpu/vpu_fw_imx6*.bin;
+	if [ $? = 0 ];then
+		file=`ls external/linux-firmware-imx/firmware/vpu/vpu_fw_imx6*.bin`
+		firmware_list="$firmware_list $file"
+	fi
+	ls external/linux-firmware-imx/firmware/vpu/Android.mk;
+	if [ $? = 0 ];then
+		file=`ls external/linux-firmware-imx/firmware/vpu/Android`
+		firmware_list="$firmware_list $file"
+	fi
 
 	tar czvf firmware.tar.gz $firmware_list;                                         checkResult
 fi
-- 
1.8.0

